docs/nidcpower/class.rst

```diff
@@ -6017,7 +6585,7 @@ source_mode
             +----------------+------------------+
             | Permissions    | read-write       |
             +----------------+------------------+
-            | Channel Based  | No               |
+            | Channel Based  | Yes              |
```

Comment:
There's a lot of changes here of properties going from "Channel Based  | No" to "Channel Based  | Yes". I believe these are bug fixes.

I suppose since the Python generated code is no different per our current design, this is just a documentation fix. But it may be worth noting in the CHANGELOG.

Reply:
Added